elasticsearch/includes/classes/class-elasticsearch.php

```diff
@@ -143,6 +143,8 @@ public function filter__ep_do_intercept_request( $request, $query, $args, $failu
 
 		$timeout = $this->get_http_timeout_for_query( $query );
 
+		// Add custom headers to identify authorized traffic
+		$args['headers'] = array_merge( $args['headers'], array( 'X-Client-Site-ID' => get_current_blog_id(), 'X-Client-Env' => VIP_GO_ENV ) );
```

Comment:
Site ID should be the Go site ID not the WordPress site ID.

Reply:
7bb95bc2 if I got that right

Reply:
Oh, actually, I guess we should technically check that `$args['headers']` is an array first.